### ISSUE TITLE
Clarify unit use of strategic resource

### ISSUE THREAD

**Issue by RealGrouchy** [yairm210/Unciv#1964]
It took me a while to figure out that the game *consumes* resources when a unit/improvement is built that requires it. I used to play Civ 3 and Civ 5 and in at least one of those I just had to have a resource available (eg horses) and I could build as many units (eg horsemen) as I wanted.

On my first game, I was quite surprised to find that I had no more coal left and thought I must have traded it away or something. Some googling and I eventually figured it out.

I have some suggestions to make this easier to understand:

1. In Civlopedia and in the construction queue info box, write "requires 1 coal" instead of "requires coal"

2. In the construction selection, where it says "requires coal" in red if there is not enough resource, keep this text if there is enough of the resource but display it differently (eg in green, or just write "1 coal" without "requires")

3. When upgrading a unit to a unit that requires a resource (eg from Artillery upgrades to Rocket Artillery which requires Aluminium), on the unit upgrade action, write "240 gold, 1 aluminium" as the cost instead of just "240 gold".

4. The strategic resource info bar at the top of the screen shows the number of a resource that is currently available. It would be helpful to have a second number in brackets after this showing how many items are currently under construction that will use this resource. Clicking on the strategic resource bar should take me to the "overview: resources" page.

5. In the Civlopedia entry for each strategic resource, list which items use that resource. In the Civlopedia entry for "Tutorials: strategic resources" add text that specifies that each building/unit that requires a resource will consume 1 of that resource.

(This is my final post for today. Thanks for listening!)

**Comment by krijnsent** [comment]
And 4b: Add to Overview->Resources a list which units consume the resources and/or Overview->Units an extra column with upkeep gold&resources

**Comment by rh-github-2015** [comment]
I always found this not easy, but OK - overview gives you a sum of resources used by units/buildings, and as there actually aren't that many objects using strategic resources, with a little brain use, it gets clear enough. But this gives me an idea - a drilldown for those rows perhaps? I just added something related in my private fork, so I think I could try. A column per unit would be trivial for resources, though IMHO not worthwhile - after all, it's a property of the base unit, not the individual, and overview->units concerns itself with instance-specific properties. Likewise gold upkeep - this is not calculated per-unit but globally. Then again, I learned that base upkeep is 0.5 from the code, not the civilopedia...
